test/client/client_test.dart

```diff
@@ -177,6 +177,7 @@ void main() {
     expect(controller.client.sendRequest('foo', {'param': 'value'}),
         throwsA(predicate((exception) {
       expect(exception, TypeMatcher<json_rpc.RpcException>());
+      exception as json_rpc.RpcException;
```

Comment:
Any reason for this line? I'm guessing this is so we can infer `exception is json_rpc.RpcException` after this point?

Reply:
That was the purpose of the cast, I went ahead and pushed a commit to use `TypeMatcher.having` instead.